winfilewriter.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"os"
+)
+
+type winFileWriter struct {
+	file *os.File
+}
+
+func NewWinFileWriter(file *os.File) *winFileWriter {
+	return &winFileWriter{
+		file: file,
+	}
+}
+
+func (wfw *winFileWriter) Write(p []byte) (n int, err error) {
+	n, err = wfw.file.Write(p)
+
+	wfw.file.Sync()
```

Comment:
I wonder how much would this impact performance on high load with a slow disk. I guess that this won't affect the perf of db provision and bindings in a noticeable way.